openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py

```diff
@@ -207,6 +253,36 @@ def test_save_application_review_info(request, user_application):
     assert updated_application.reviewed_by == request.user
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'status, expected_email_template', [
+        (UserApplication.WAITLIST, MandrillClient.APPLICATION_WAIT_LISTED),
+        (UserApplication.ACCEPTED, MandrillClient.APPLICATION_ACCEPTED)
+    ], ids=['wait_list', 'accepted']
+)
+@mock.patch('openedx.adg.lms.applications.admin.get_user_first_name', return_value='test')
+@mock.patch('openedx.adg.lms.applications.admin.MandrillClient.__init__', return_value=None)
+@mock.patch('openedx.adg.lms.applications.admin.MandrillClient.send_mandrill_email')
+def test_send_acceptance_wait_list_email(
+    mock_send_mandrill_email, mock_init, mock_get_first_name, status, expected_email_template, request, user_application
+):
+    """ Test that on acceptance and wait list decision user is informed by an email with correct template """
```

Comment:
*"Test that upon application status update, applicant is intimated via an email with correct template and context"